### PR TITLE
Updated JavaScript Pages

### PR DESCRIPTION
**Pull request by WaffleCohn**

### CODE REVIEW

File: javascript/index.md [hunk 1]
```diff
@@ -23,28 +23,31 @@ JavaScript tasks.
...
-1. [JavaScript Lab]({{ site.baseurl }}/javascript/lab/)
+1. [Introduction to JavaScript Slides](https://wafflecohn.github.io/javascript-lab/intro-javascript-slides.pdf)
 1. [HTML & CSS Recap]({{ site.baseurl }}/javascript/html-css/)
-1. [Demo of Final Project]({{ site.baseurl }}/javascript/demo/)
-1. [Source code of Final Project](https://github.com/jez/jquery-lab/)
+1. [JavaScript Lab]({{ site.baseurl }}/javascript/lab/)
+1. [Demo of Final Project](https://wafflecohn.github.io/javascript-lab/)
+1. [Source code of Final Project](https://github.com/WaffleCohn/javascript-lab)
+1. [Old Introduction to JavaScript]({{ site.baseurl }}/javascript/js-intro/)
+1. [Old JavaScript Lab]({{ site.baseurl }}/javascript/lab/oldlab.html)
```
Comment (serena-w) [inline javascript/index.md]: @skrulcik this should just be `{{ site.baseurl }}/javascript/lab/oldlab`, right?

Comment (skrulcik) [reply javascript/index.md]: Well the way this is structured now, `.html` is the way to go. But most of the site would move `oldlab.md` to `oldlab/index.md`, which allows us to drop the filename extension like you describe.

To stay consistent across the repo, Ari can you move to the `index.md` structure so we can keep all links extensionless?

Comment (WaffleCohn) [reply javascript/index.md]: Sure

**Review by skrulcik — APPROVED** [review]
LGTM, I'll let Serena give this final approval and merge it

**Pull request merged**